File: Java/src/com/postgres/Book.java
package com.postgres;

public class Book {
  private String isbn;
  private String title;
  private String author;
  private int publicationYear;

  public Book(String isbn, String title, String author, int publicationYear) {
    this.isbn = isbn;
    this.title = title;
    this.author = author;
    this.publicationYear = publicationYear;
  }

  public String getIsbn() {
    return isbn;
  }

  public String getTitle() {
    return title;
  }

  public String getAuthor() {
    return author;
  }

  public int getPublicationYear() {
    return publicationYear;
  }
}

File: Go/models/book.go
package models

type Book struct {
	isbn            string
	title           string
	author          string
	publicationYear int
}

func NewBook(isbn string, title string, author string, publicationYear int) *Book {
	return &Book{
		isbn:            isbn,
		title:           title,
		author:          author,
		publicationYear: publicationYear,
	}
}

func (b *Book) GetIsbn() string {
	return b.isbn
}

func (b *Book) GetTitle() string {
	return b.title
}

func (b *Book) GetAuthor() string {
	return b.author
}

func (b *Book) GetPublicationYear() int {
	return b.publicationYear
}
